config/biggerreactors-server.toml

```diff
@@ -2,7 +2,13 @@
 # Enabled advanced config options
 # Additional options will be shown after next config load
 #
-EnableAdvancedConfig = false
+EnableAdvancedConfig = true
```

Comment:
not documented anywhere (as per usual, im bad at that whole _documenting things_ thing), you dont need this to be able to use the values, it will read them and leave them in the config file, if it doesnt, thats a bug. that functionality is mostly aimed at allowing pack devs to ship a minimal config with only the edits they made saved in VCS.

Reply:
So would I just manually add the settings in to change them (just copy paste from Advanced Config) or should I switch to advanced, save the edits to the advanced config, then switch back to false?